lib/api/assertions/hasAttribute.js

```diff
@@ -36,31 +36,23 @@ exports.assertion = function(definition, expectedAttribute, msg) {
  
     return {
       message,
-      args: [this.elementSelector, `'${Array.isArray(expectedAttribute) ? expectedAttribute.join(' ') : expectedAttribute}'`]
+      args: [this.elementSelector, `'${expectedAttribute}'`]
```

Comment:
I don't think `hasAttribute` will be able to support multiple attribute as refered in our docs. We need to change our [docs](https://nightwatchjs.org/api/assert/#assert-hasAttribute). Also add a warning this is not supported from now

Reply:
IMO, adding a warning isn't useful as no one is using this API because it wasn't functional. But updating the docs is necessary to avoid confusion for future users.

Reply:
Users can use the assertion by supplying an array of attributes. We would need a check if it's a single attribute or if the function has been provided with multiple attributes.

Reply:
Added a parameter check for `expectedAttribute` 👍